src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -948,7 +912,7 @@ struct FollowPath
         }
         cmds = SHAPE(sh)->rs.path.cmds.data;
         cmdsCnt = SHAPE(sh)->rs.path.cmds.count;
-        totalLen = _pathLength(cmds, cmdsCnt, pts, SHAPE(sh)->rs.path.pts.count);
+        totalLen = tvg::pathLength(cmds, cmdsCnt, pts, SHAPE(sh)->rs.path.pts.count);
```

Comment:
Please consider this interface:
```
totalLen =  sh->length();  or SHAPE(sh)->length();
```

Reply:
in the tvgTrimPath I'll still need `tvg::length(cmds, cmdsCnt, pts, ptsCnt)`. so `SHAPE(sh)->length()` would simply call `tvg::length` - do I understand correctly?